src/test/e2e/30_config_file_test.go

```diff
@@ -137,6 +140,7 @@ func configFileDefaultTests(t *testing.T) {
 
 	// Test remaining default initializers
 	os.Setenv("ZARF_CONFIG", filepath.Join("src", "test", "zarf-config-test.toml"))
```

Comment:
Should be able to set these with `t.Setenv` , not a priority though

Reply:
fixed